agents/holo_fpga/roach_agent.py

```diff
@@ -0,0 +1,182 @@
+import argparse
+import os
+import numpy as np
+import txaio
+import yaml
+import time
+
+ON_RTD = os.environ.get("READTHEDOCS") == "True"
+if not ON_RTD:
+    import casperfpga
+    from holog_daq import fpga_daq3, synth3
+    from ocs import ocs_agent, site_config
+    from ocs.ocs_twisted import TimeoutLock
```

Comment:
Putting ocs in this `ON_RTD` protection has the potential to cause issues with the docs builds if the `@params` decorator is ever used, best to move it out of (and above) this `if` statement.